.kubernetes/kubernetes_ingresses.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+  name: fakebook
+spec:
+  rules:
+  - host: fakebook.revaturelabs.com
+    http:
+      paths:
```

Comment:
@escalonn did you mean to leave out `fakebook-profile-api` from the Ingress definition?